uat/src/main/resources/greengrass/features/log-manager-1.feature

```diff
@@ -0,0 +1,187 @@
+@LogManager
+Feature: Greengrass V2 LogManager
+
+    As a customer, I want to selectively upload my logs and metrics to AWS Cloudwatch to save cost.
```

Comment:
Delete add a dummy tests to proof the setup works

Reply:
Here is an example of a dummy test implemented on the nucleus project

Reply:
added